docs/witx/typenames.witx

```diff
@@ -44,7 +44,7 @@
     ;;; Argument list too long.
     $2big
     ;;; Permission denied.
-    $acces
+    $access
```

Comment:
This should be ignored: https://github.com/WebAssembly/WASI/blob/main/phases/snapshot/witx/typenames.witx#L443-L455